core/capabilities/ccip/configs/evm/contract_reader.go

```diff
@@ -113,6 +113,16 @@ var DestReaderConfig = evmrelaytypes.ChainReaderConfig{
 					ChainSpecificName: mustGetEventName(consts.EventNameExecutionStateChanged, offrampABI),
 					ReadType:          evmrelaytypes.Event,
 				},
+				//nolint:staticcheck // TODO: remove deprecated config.
+				consts.MethodNameOfframpGetStaticConfig: {
```

Comment:
This is duplicated, I see it on line 96 as well

Reply:
Most places seem to use capitol r in OffRamp/OnRamp, I'm fixing that. After this change I'll update `chainlink-ccip` to use the new name, and after that change I'll remove these deprecated configs.